src/components/Composer/index.js

```diff
@@ -86,387 +85,336 @@ const propTypes = {
     ...windowDimensionsPropTypes,
 };
 
-const defaultProps = {
-    defaultValue: undefined,
-    value: undefined,
-    numberOfLines: undefined,
-    onNumberOfLinesChange: () => {},
-    maxLines: -1,
-    onPasteFile: () => {},
-    shouldClear: false,
-    onClear: () => {},
-    style: null,
-    isDisabled: false,
-    autoFocus: false,
-    forwardedRef: null,
-    onSelectionChange: () => {},
-    selection: {
-        start: 0,
-        end: 0,
-    },
-    isFullComposerAvailable: false,
-    setIsFullComposerAvailable: () => {},
-    isComposerFullSize: false,
-    shouldCalculateCaretPosition: false,
-    checkComposerVisibility: () => false,
-};
+// Get characters from the cursor to the next space or new line
+const getNextChars = (str, cursorPos) => {
```

Comment:
Should we be moving this method to another file?

Reply:
True, we currently lack text or string utilities. @mananjadhav, do you think it would be beneficial to create a new utility for this purpose?

Reply:
I think that would be handy but we can make it in a follow up PR